lib/screens_web/views/v2/audio/departures_view.ex

```diff
@@ -17,6 +17,11 @@ defmodule ScreensWeb.V2.Audio.DeparturesView do
     ~E|<p><%= text %></p>|
   end
 
+  defp render_section(%{type: :normal_section, header: header, departure_groups: departure_groups})
+       when is_binary(header) do
+    ~E|<p><%= header %></p><%= Enum.map(departure_groups, &render_departure_group/1) %>|
+  end
```

Comment:
**note:** I wasn't sure _how_ the header should be rendered for audio but this made sense to me.

Reply:
Seems reasonable. We will get a better feel for it once we do the full audio equivalence implementation for this screen type.